lib/order.rb

```diff
@@ -0,0 +1,28 @@
+class Order
+  attr_reader :menu, :dishes, :quantity, :total
+
+  def initialize(menu_klass)
+    @menu = menu_klass
+    @dishes = []
+  end
+
+  def choose(dish, quantity=1)
+    quantity.times do
+      menu.select { |k,v| dishes.push([k, v]) if k == dish }
+    end
+  end
+
+  def calculate_quantities(order)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `order`. If it's necessary, use `_` or `_order` as an argument name to indicate that it won't be used. You can also write as `calculate_quantities(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)